### ISSUE TITLE
Show what has happened since last login

### ISSUE THREAD

**Issue by gorkem-cetin** [Countly/countly-server#120]
Between two consecutive logins, Countly can show: 
- Number of sessions
- Number of users
- New countries
- Change in top operator
- Change in top platform
- Number of new device(s) added (optional, may not be vital)

Initially, general set of rules would be: 

If ($diff_days > 3), print "We missed you!" 

if ($users) > {
print "Over the last $diff_days days you had $sessions sessions, generated by $users users, an average of $sessions/$diff_days per day ((if $countries>0) from $countries countries), {if ($new_devices>0) adding $new_devices devices}.
}

if $(top_operator_changes) or $(top_platform_changes) {
print "Moreover, {if ($top_operator_changes) your top operator is now $top_operator} and {if ($top_platform_changes) top platform is $top_platform}.
}

**Comment by gorkem-cetin** [comment]
I think we can work on it Weite for next release.

**Comment by ar2rsawseen** [comment]
@waiterZen btw, we now have last_login timestamp for each user from the core, no need to add it separately in plugin :)

**Comment by ar2rsawseen** [comment]
Has this been added in Assistant? @ArtursKadikis if yes, you can close it

**Comment by gorkem-cetin** [comment]
Basically we have discussed that (dont remember when and with whom, interestingly), this tas was a reason that we started Assistant.. But then requirements list grew and this task became a minor part of what Assistant could be doing. It hasn't been added yet as far as I know, but I believe we should discuss in detail whether it should really be added or not, preferably after 17.04 release. So I suggest that it is kept open for now and we can revisit later, depending on our task load and future todo items.
